lib/kubernetes-deploy/kubernetes_resource/bugsnag.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class Bugsnag < KubernetesResource
+    TIMEOUT = 1.minute
+
+    def initialize(name, namespace, context, file)
+      @name = name
+      @namespace = namespace
+      @context = context
+      @file = file
+      @secret_found = false
+    end
+
+    def sync
+      _, _err, st = run_kubectl("get", type, @name)
+      @found = st.success?
+      if @found
+        secrets, _err, st = run_kubectl("get", "secrets", "--output=jsonpath='{.items[*].metadata.name}'")
+        @secret_found = secrets.split.any?{|s| s.end_with?("-bugsnag")}
```

Comment:
I'm still unsure what exactly should the secret name be, so this just looks if we have any bugsnag secret. Should be fine as long as apps don't start having multiples (unclear why would that be desirable)